diff --git a/node_modules/react-native-select-contact/android/build.gradle b/node_modules/react-native-select-contact/android/build.gradle
index 65aa1d7..c2145da 100644
--- a/node_modules/react-native-select-contact/android/build.gradle
+++ b/node_modules/react-native-select-contact/android/build.gradle
@@ -43,7 +43,7 @@ repositories {
         url "$rootDir/../node_modules/jsc-android/dist"
     }
     google()
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {
